pkgs/development/interpreters/php/default.nix

```diff
@@ -11,6 +11,8 @@ let
 
     let php7 = lib.versionAtLeast version "7.0";
         mysqlHeaders = mysql.lib.dev or mysql;
+        mysqlndSupport = config.php.mysqlnd or false;
+        mysqlBuildInputs = if mysqlndSupport then [] else [ mysqlHeaders ];
```

Comment:
-        mysqlBuildInputs = if mysqlndSupport then [] else [ mysqlHeaders ]; 
+       mysqlBuildInputs = lib.optional mysqlndSupport mysqlHeaders;

Reply:
You've reversed the conditional.

Reply:
Applied in 1cff740